lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -103,9 +103,7 @@ def get_certificate(self, model):
         certificate_info = certificate_downloadable_status(model.user, model.course_id)
         if certificate_info['is_downloadable']:
             return {
-                'url': self.context['request'].build_absolute_uri(
-                    certificate_info['download_url']
-                ),
+                'url': '',
```

Comment:
Interesting)
This field method looks pretty useless, isn't it?) I mean it returns {} or {url: ''}

I'm curious if we could get rid of it